### DISCUSSION TITLE
Can I do shades at 50%? A middle point?

### DISCUSSION THREAD

**Discussion by lacedup** [aarons22/homebridge-bond#161] [Q&A]
Instead of full up or down?

**Comment by aarons22** [comment]
I think this can be done with a Preset?

![IMG_F3F081FFA7FB-1](https://user-images.githubusercontent.com/1285320/136869074-8848eef6-f4a5-4c70-91b6-1a4daa8aa58a.jpeg)

**Reply by aarons22** [reply]
Otherwise, there isn't much else this plugin can do with the current Bond API. If you want more control on your blinds, I'd recommend checking out this plug-in: https://www.npmjs.com/package/homebridge-blinds
